src/player/Player.php

```diff
@@ -1711,7 +1719,9 @@ public function useHeldItem() : bool{
 		$this->resetItemCooldown($oldItem);
 		$this->returnItemsFromAction($oldItem, $item, $returnedItems);
 
-		$this->setUsingItem($item instanceof Releasable && $item->canStartUsingItem($this));
+		$shouldStartUsing = ($item instanceof Releasable && $item->canStartUsingItem($this)) ||
+			($item instanceof Chargeable && $result !== ItemUseResult::SUCCESS && $item->canStartChargingItem($this));
```

Comment:
I don't like this. We could just as easily implement Releasable on Crossbow and achieve the exact same result